[{"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/MI_LakeErie_UTM17_2010/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/MN_FullState/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/MO_JeffersonGenevieve_2010/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/MO_JeffersonGenevieve_2010/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/MO_WestCentral_4_2018/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/MS_CampShelby_2006/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/MS_CampShelby_2006/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NC_HurricaneFlorence_2_2020/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NC_HurricaneFlorence_2_2020/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NC_Phase5_McDowell_2017/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NC_Phase5_McDowell_2017/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NE_Niobrara_Topobathy_2018/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NE_Northeast_3_2020/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NE_Northeast_3_2020/ept-data/1-1-0-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_SouthCentral_B1_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_SouthCentral_B1_2018/ept-data/1-1-0-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NY_3County_2019/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NY_3County_2019/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SC_CherokeeCo_2008/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/SC_CherokeeCo_2008/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SC_SavannahPeeDee_2_2019/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/SC_SavannahPeeDee_2_2019/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_James_River_NRCS_G_2010/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_James_River_NRCS_G_2010/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/TX_Coastal_B2_Z15_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/TX_Coastal_B2_Z15_2018/ept-data/0-0-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AL_25Co_B3_2017/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AL_25Co_B3_2017/ept-data/1-0-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AR_LAnguillee_2011_LAS_2016/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AR_LAnguillee_2011_LAS_2016/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AZ_BrawleyRillito_FEMA_2018/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AZ_BrawleyRillito_FEMA_2018/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_LakeCounty_2015_LAS_2017/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_LakeCounty_2015_LAS_2017/ept-data/1-1-1-0.laz\n\n"}, {"error": "Could not create geometry because of errors while reading input."}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_NM_NRCS_Central_A3_2017_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_NM_NRCS_Central_A3_2017_LAS_2019/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_PR_PRVI_C_2018/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_PR_PRVI_C_2018/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_SD_MORiver_Woolpert_B2_2016/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_SD_MORiver_Woolpert_B2_2016/ept-data/1-1-1-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_Lower_SanBernard_B6_2017_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_Lower_SanBernard_B6_2017_LAS_2019/ept-data/2-3-1-2.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_Panhandle_B9_2017_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_Panhandle_B9_2017_LAS_2019/ept-data/1-1-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_South_B8_2018_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_South_B8_2018_LAS_2019/ept-data/1-0-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_UT_Central_QL1_B1_2018_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_UT_Central_QL1_B1_2018_LAS_2019/ept-data/2-2-0-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_WA_GlacierPeak_2014_LAS_2016/ept.json"], "error": ""}, {"tile": "USGS_LPC_WA_Western_North_2016_LAS_2018", "error": "Expecting value: line 1 column 1 (char 0)"}, {"tile": "USGS_LPC_WA_Western_South_2016_LAS_2018", "error": "Expecting value: line 1 column 1 (char 0)"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/US_MexicanBorder_UTM14_2007/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/US_MexicanBorder_UTM14_2007/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/WA_EasternCascades_5_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/WA_EasternCascades_5_2019/ept-data/1-0-0-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_Jefferson_2019/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_Jefferson_2019/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_Oshkosh_3Rivers_FondDuLac_TL_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_Oshkosh_3Rivers_FondDuLac_TL_2018/ept-data/0-0-0-0.laz\n\n"}]